pkg/controlplane/instance.go

```diff
@@ -166,11 +168,10 @@ func (cp *Instance) GetAllPeers() []*cpstore.Peer {
 // CreateExport defines a new route target for ingress dataplane connections.
 func (cp *Instance) CreateExport(export *cpstore.Export) error {
 	cp.logger.Infof("Creating export '%s'.", export.Name)
-	exSvc := export.ExportSpec.ExternalService
-	if (exSvc.Host != "" && exSvc.Port == 0) || (exSvc.Host == "" && exSvc.Port != 0) {
-		return fmt.Errorf("ExternalService (Host: %s ,Port: %d) wasn't set properly", exSvc.Host, exSvc.Port)
+	eSpec := export.ExportSpec
+	if eSpec.ExternalService != "" && !valid.IsIP(eSpec.ExternalService) && !valid.IsDNSName(eSpec.ExternalService) {
```

Comment:
nit: is there a way to validate these without adding a package? For example, [net.ParseIP](https://pkg.go.dev/net#example-ParseIP) and actually resolving the DNS name if it's not an IP?

Reply:
Done, use an internal function.